File: reDiscover/TGAppDelegate.m
//
//  TGAppDelegate.m
//  Proto3
//
//  Created by Teo Sartori on 13/03/2013.
//  Copyright (c) 2013 Teo Sartori. All rights reserved.
//

#import "TGAppDelegate.h"
#import "TGMainViewController.h"
#import "TGWindowController.h"

@implementation TGAppDelegate

- (void)applicationDidFinishLaunching:(NSNotification *)aNotification
{


    if (_theWindowController == NULL) {
        _theWindowController = [[TGWindowController alloc] initWithWindowNibName:@"TGWindowController"];
    }
    [_theWindowController showWindow:self];    
}

- (void)applicationWillTerminate:(NSNotification *)notification {
    NSLog(@"reDiscover is quitting");
}
@end

File: source/TGAppDelegate.swift
//
//  TGAppDelegate.swift
//  reDiscover
//
//  Created by Teo on 09/02/15.
//  Copyright (c) 2015 Teo Sartori. All rights reserved.
//

import Cocoa

class TGAppDelegate: NSObject, NSApplicationDelegate {
    
    func applicationDidFinishLaunching(_ aNotification: Notification) {
        // Insert code here to initialize your application
//        println("We did \(self.window)")
        NSApplication.shared()
    }
    
    func applicationWillTerminate(_ aNotification: Notification) {
        // Insert code here to tear down your application
    }
    
    func applicationDidBecomeActive(_ notification: Notification) {
        print("Active application!")
    }
}
